github/actions_workflow_runs_test.go

```diff
@@ -31,11 +31,11 @@ func TestActionsService_ListWorkflowRunsByID(t *testing.T) {
 		t.Errorf("Actions.ListWorkFlowRunsByID returned error: %v", err)
 	}
 
-	want := &Runs{
+	want := &WorkflowRuns{
 		TotalCount: Int(4),
-		Runs: []*WorkflowRun{
-			{ID: Int64(399444496), RunNumber: Int64(29679449), CreatedAt: &Timestamp{time.Date(2019, time.January, 02, 15, 04, 05, 0, time.UTC)}, UpdatedAt: &Timestamp{time.Date(2020, time.January, 02, 15, 04, 05, 0, time.UTC)}},
-			{ID: Int64(399444497), RunNumber: Int64(29679449), CreatedAt: &Timestamp{time.Date(2019, time.January, 02, 15, 04, 05, 0, time.UTC)}, UpdatedAt: &Timestamp{time.Date(2020, time.January, 02, 15, 04, 05, 0, time.UTC)}},
+		WorkflowRuns: []*WorkflowRun{
+			{ID: Int64(399444496), RunNumber: Int(29679449), CreatedAt: &Timestamp{time.Date(2019, time.January, 02, 15, 04, 05, 0, time.UTC)}, UpdatedAt: &Timestamp{time.Date(2020, time.January, 02, 15, 04, 05, 0, time.UTC)}},
```

Comment:
nit: for the `RunNumber` values, let's kick this down to maybe `296` or another relatively small number so that we don't give the impression that these are IDs from a quick glance.